xml/ay_auto_installation.xml

```diff
@@ -838,44 +838,35 @@ default</screen>
     <title>Combining the <command>linuxrc</command> <filename>info</filename> File with the &ay; Control File</title>
     <para>
      If you choose to pass information to <command>linuxrc</command> using the
-     <filename>info</filename> file, it is possible to integrate the
-     keywords in the XML control file. In this case the file needs to be
-     accessible to <command>linuxrc</command> and must be named
-     <filename>info</filename>.
+     <filename>info</filename> file or as boot options, you may integrate the
+     keywords into the &ay; control file. Add an <literal>info_file</literal> 
+     section as shown in the example below. This section contains 
+     keyword - value pairs, separated by colons, one pair per line.
     </para>
     <para>
-     The <command>linuxrc</command> program will look for the string
-     <literal>start_linuxrc_conf</literal> in the &ay; control file, which
-     represents the beginning of the embedded <command>linuxrc</command> control
-     file. If the string is found, <command>linuxrc</command> will parse the
-     content starting from that string and will finish when the string
-     <literal>end_linuxrc_conf</literal> is found. The options are stored in the
-     control file in the following way:
+
     </para>
     <example>
      <title><command>linuxrc</command> Options in the &ay; Control File</title>
-<screen>....
-  &lt;install&gt;
+<screen><![CDATA[....
+  <install>
 ....
-    &lt;init&gt;
-      &lt;info_file&gt;
-&lt;![CDATA[
+    <init>
+      <info_file>
 
-install: nfs://192.168.1.1/CDs/full-i386
+install: https://download.opensuse.org/tumbleweed/repo/oss
```

Comment:
Do we want to use *openSUSE* in the example? I recently closed a different Bugzilla specifically to remove such an example...